main.py

```diff
@@ -8,7 +8,7 @@
 from dotenv import load_dotenv
 
 # Setup connection to database
-con = sqlite3.connect('irishl.db')
+con = sqlite3.connect('copy.db')
```

Comment:
Probably don't want this change in the PR, I assume you had changed this just for local testing?
To avoid this in the future we can add it as an environment variable if you like